manual/C/ch_Finance-Quote.xml

```diff
@@ -216,9 +218,9 @@ SYNOPSIS
       <step>
         <simpara>Install &app-xcode;
           <footnote>
-            <para>&app-xcode; is an optional tool from your
-              &mac; distribution DVD, but also at <ulink url="&url-ap-dev;xcode/" />
-              the application is also available.
+            <para>You can install &app-xcode; from the App Store or install the
+              much smaller Xcode command line tools by running <userinput>
+              sudo xcode-select --install</userinput> from a Terminal.app prompt.
```

Comment:
I think I would mention hat xcode contains devel tools, which are required by CPAN to build and test your desired modules.